sdk/src/test/java/com/mapbox/maps/ViewAnnotationManagerTest.kt

```diff
@@ -36,12 +34,14 @@ class ViewAnnotationManagerTest {
     every { view.visibility } returns View.VISIBLE
     every { view.viewTreeObserver } returns mockk(relaxed = true)
     every { mapView.context } returns mockk()
+    every { view.addOnAttachStateChangeListener(any()) } just Runs
     every { mapboxMap.addViewAnnotation(any(), any()) } returns ExpectedFactory.createNone()
   }
 
   @After
   fun tearDown() {
     every { mapboxMap.removeViewAnnotation(any()) } returns ExpectedFactory.createNone()
+    every { view.removeOnAttachStateChangeListener(any()) } just Runs
```

Comment:
Shouldn't be included verification of this call:
`inflatedView.viewTreeObserver.removeOnGlobalLayoutListener` ?

Reply:
as current test setup is built - I guess not 😄